### ISSUE TITLE
Spacemacs not loading font on OSX

### ISSUE THREAD

**Issue by afabbro** [syl20bnr/spacemacs#2864]
I'm pretty confident that I'm followed the instructions for the Powerline-patched fonts. They're on my system and available in other applications.

However, I get:

(Spacemacs) Warning: Cannot find font "Sauce Code Powerline"! 

when running Spacemacs and it hits the default font setting line.

Where does spacemacs (and/or powerline) look for the fonts?

**Comment by d12frosted** [comment]
Are you trying it in terminal? Then you need to set it in terminal preferences for your current profile.

**Comment by timstott** [comment]
Facing identical issue here. I'm using spacemacs from the terminal (iTerm2 | OS X).

![screen shot 2015-10-16 at 10 57 00](https://cloud.githubusercontent.com/assets/1334474/10539039/acbe49a6-73f4-11e5-8235-b928bedbed7c.png)

```
;; .spacemacs
dotspacemacs-default-font '("Droid Sans Mono for Powerline"
                            :size 12
                            :weight normal
                            :width normal
                            :powerline-scale 1.3)  
```

@d12frosted any thoughts? :cake:

**Comment by d12frosted** [comment]
@timstott 
In the terminal this warning is expected. Just ignore it. :smile:

**Comment by StreakyCobra** [comment]
We should probably document somewhere that the font in terminal mode is set by the terminal and can not be changed by emacs.

**Comment by d12frosted** [comment]
I would vote for both.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
